components/PlantForm.js

```diff
@@ -0,0 +1,206 @@
+        ></textarea>
+
+        <label htmlFor="lightNeed">
+          <h3>Light Need: *</h3>
+        </label>
+
+        <section>
+          <div>
+            <input
+              type="radio"
+              id="lightNeed1"
+              name="lightNeed"
+              value="full sun"
+              required
+            />
+            <label htmlFor="lightNeed1">Full Sun</label>
+          </div>
+          <div>
+            <input
+              type="radio"
+              id="lightNeed2"
+              name="lightNeed"
+              value="partial shade"
+            />
+            <label htmlFor="lightNeed2">Partial Shade</label>
+          </div>
+          <div>
+            <input
+              type="radio"
+              id="lightNeed3"
+              name="lightNeed"
+              value="full shade"
+            />
+            <label htmlFor="lightNeed3">Full Shade</label>
+          </div>
+        </section>
+
+        <label htmlFor="waterNeed">
+          <h3>Water Need: *</h3>
+        </label>
+
+        <section>
+          <div>
+            <input
+              type="radio"
+              id="waterNeed1"
+              name="waterNeed"
+              value="low"
+              required
+            />
+            <label htmlFor="waterNeed1">Low</label>
+          </div>
+          <div>
+            <input
+              type="radio"
+              id="waterNeed2"
+              name="waterNeed"
+              value="medium"
+            />
+            <label htmlFor="waterNeed2">Medium</label>
+          </div>
+          <div>
+            <input type="radio" id="waterNeed3" name="waterNeed" value="high" />
+            <label htmlFor="waterNeed3">High</label>
+          </div>
+        </section>
+
+        <label htmlFor="fertiliserSeason">
+          <h3>Fertiliser Season:</h3>
+        </label>
+
+        <section>
+          <div>
+            <input
+              type="checkbox"
+              id="fertiliserSeason1"
+              name="fertiliserSeason"
+              value="spring"
+            />
+            <label htmlFor="fertiliserSeason1">Spring</label>
+          </div>
+          <div>
+            <input
+              type="checkbox"
+              id="fertiliserSeason2"
+              name="fertiliserSeason"
+              value="summer"
+            />
+            <label htmlFor="fertiliserSeason2">Summer</label>
+          </div>
+          <div>
+            <input
+              type="checkbox"
+              id="fertiliserSeason3"
+              name="fertiliserSeason"
+              value="autumn"
+            />
+            <label htmlFor="fertiliserSeason3">Autumn</label>
+          </div>
+          <div>
+            <input
+              type="checkbox"
+              id="fertiliserSeason4"
+              name="fertiliserSeason"
+              value="winter"
+            />
+            <label htmlFor="fertiliserSeason3">Winter</label>
```

Comment:
Fix the duplicate htmlFor attribute in the "Winter" checkbox.
```suggestion
            <label htmlFor="fertiliserSeason4">Winter</label>
```